### ISSUE TITLE
Add in the email Subject box a short text for the WARNING/FAILURE reasons 

### ISSUE THREAD

**Issue by svoinea** [informatics-isi-edu/iobox-win32#23]
The Subject box from the email should contain a short text description for the WARNING/FAILURE reasons.
Example of such texts:

```
 - No output directory
 - No credentials file
 - Invalid number of rows returned
 - Duplicate found
 - Can not create namespace
```

**Comment by svoinea** [comment]
Fixed in the commit 10d2077cb7e831d8aead69bd447dacd607b61312

**Comment by svoinea** [comment]
Ready for review.
